scripts/support/README.rst

```diff
@@ -45,6 +45,9 @@ the support account, encrypt the password and print the encrypted password to co
 Installation
 ============
 
+Some scripts of this folder use *osclients.py*. That file can be copied from
+*utils/osclients.py*.
```

Comment:
The scripts to create the base image and insert the support script are independent of the glancesync project (the installation and use instructions are independent, for example), but the osclients.py file is the same. This line only says where the file is. I think that is better this than having three copies of the file.